### PR TITLE
Easier skimming of ventilation section

### PR DESCRIPTION
**Pull request by catherio**

### CODE REVIEW

File: src/posts/paper/14-research-sources.ts [hunk 1]
```diff
@@ -91,12 +93,18 @@ if height = 8ft,
 CFM = 2/3 * (square footage)
 \`\`\`
 
-From this, we also deduce that areas that have high rates of air exchange while not being "outside" per-se should be similarly safer. This includes cars with the windows rolled down and partially enclosed spaces (i.e. at least one wall is open to outside air).
+###### Moving car with windows rolled down; a space with one or more sides open to the outdoors
```
Comment (beshaya) [inline src/posts/paper/14-research-sources.ts:96]: Perhaps easier on the eyes / easier to skip if this is two sections separate sections?

**Review by beshaya — CHANGES_REQUESTED** [review]

File: src/posts/paper/14-research-sources.ts (hunk 1 above)
Comment (blanchardjeremy) [reply src/posts/paper/14-research-sources.ts]: @beshaya I went ahead and separated them. Whatcha think?

Comment (jy2wong) [reply src/posts/paper/14-research-sources.ts]: I like how it looks separated!

**Comment by catherio** [comment]
I don't care anymore 😅

**Pull request closed without merging**